src/main/java/TaskList.java

```diff
@@ -0,0 +1,26 @@
+import java.util.LinkedList;
+
+public class TaskList {

```

Comment:
Personally, I use TaskList to do a lot of processing and also the generation of reports (formatted text to be stored inside the data.txt file and output list).